docs/gathered-data.md

```diff
@@ -52,17 +52,29 @@ The CRD sizes above are in the raw (uncompressed) state.
 
 ## CephCluster
 
-collects statuses of the`cephclusters.ceph.rook.io` resources
+Collects statuses of the`cephclusters.ceph.rook.io` resources
 from Openshift Data Foundation Stack.
 
-API Reference:
+### API Reference:
+https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
 
-	https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
+### Sample data:
+docs/insights-archive-sample/config/storage/openshift-storage/cephclusters/ocs-storagecluster-cephcluster.json
 
-* Location in archive: config/storage/<namespace>/<name>.json
-* Id in config: clusterconfig/ceph_cluster
-* Since versions:
-  - 4.12+
+### Location in archive:
+| Version | Path													 |
+| ------- | ----------------------------------------------------- |
+| <= 4.11 | config/storage/{namespace}/cephclusters/{name}.json 	 |
+| 4.12+   | config/storage/{namespace}/cephclusters/{name}.json 	 |
+
+### Config ID:
+clusterconfig/ceph_cluster
+
+### Since versions:
+* 4.8.24
+* 4.9.23
+* 4.10.17
```

Comment:
Are we really sure that this data is collected in "such old OCP versions" ?

Reply:
OK so this seems to go down to 4.9 see https://issues.redhat.com/browse/OCPBUGS-251. Can we verify the 4.8 then?

Reply:
https://github.com/openshift/insights-operator/pulls?q=is%3Apr+is%3Aclosed+ceph

Reply:
OK thanks :)